action-versions-updates/src/test/kotlin/io/github/typesafegithub/workflows/dsl/CheckVersionsTests.kt

```diff
@@ -0,0 +1,120 @@
+package io.github.typesafegithub.workflows.dsl
```

Comment:
```suggestion
package io.github.typesafegithub.workflows.updates
```

+ please move the file accordingly.